bower.json

```diff
@@ -31,22 +31,25 @@
   ],
   "dependencies": {
     "firebase": "^2.1.0",
-    "ember-data": "^1.13.0"
+    "ember-data": "^2.0.0"
   },
   "devDependencies": {
-    "ember": "1.13.3",
+    "ember": "^2.0.0",
```

Comment:
As far as I can see from ember-try config, only ember-data is changed, or does that automatically try the newest ember version as well? 

I will set all to fixed versions.